core/arch/arm/plat-stm32mp2/platform_config.h

```diff
@@ -0,0 +1,115 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2021, STMicroelectronics
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT			32
+
+/* SoC interface registers base address ranges */
+#define APB1_BASE			0x40000000
+#define APB1_SIZE			0x00200000
+#define APB2_BASE			0x40200000
+#define APB2_SIZE			0x00040000
+#define AHB2_BASE			0x40400000
+#define AHB2_SIZE			0x01c00000
+#define AHB3_BASE			0x42000000
+#define AHB3_SIZE			0x02000000
+#define APB3_BASE			0x44000000
+#define APB3_SIZE			0x001f0000
+#define AHB4_BASE			0x44200000
+#define AHB4_SIZE			0x01e00000
+#define SAPB_BASE			0x46000000
+#define SAPB_SIZE			0x00200000
+#define SAHB_BASE			0x46200000
+#define SAHB_SIZE			0x01e00000
+#define APB4_BASE			0x48000000
+#define APB4_SIZE			0x00200000
+#define AHB5_BASE			0x48200000
+#define AHB5_SIZE			0x01e00000
+
+/* SoC interface registers base address */
+#define I2C4_BASE			0x40150000
+#define I2C6_BASE			0x40170000
```

Comment:
These two defines are not at the right place based on their addresses.
They should be between UART5 and UART6

Reply:
not yet addressed

Reply:
Thanks for noticing, I'll change that right away